### DISCUSSION TITLE
Alignment options for script tags on GitHub pages (Jekyll chirpy)

### DISCUSSION THREAD

**Discussion by CSpanias** [11ty/eleventy#3109] [Q&A] [answered]
Hi all, 

I am trying to place two script tags next to each other, but I have failed miserably. 

I have the following two "badges" that I want to use in my [About](https://cspanias.github.io/about/) page:

```
<script src="https://www.hackthebox.com/badge/1705946"></script>  
<script src="https://tryhackme.com/badge/2134791"></script>
```

I have tried inserting them as images:
```
![](https://www.hackthebox.com/badge/1705946)
![](https://tryhackme.com/badge/2134791)
```

I have also tried different things that I found online, such as:
```
[ ![](https://www.hackthebox.eu/badge/image/1705946)](https://www.hackthebox.eu/home/users/profile/1705946)
[ ![](https://tryhackme.com/badge/2134791)](https://tryhackme.com/badge/2134791)
```

Nothing worked so far. I am might be trying to achieve something that is not realistic, so I thought to ask. I have read how to do it via CSS, but since I am not confident with web dev in general, I did not tinker around the theme files.

Thank you in advance for your time and help,

Charalampos

**Comment by Snapstromegon** [comment, marked as answer]
I personally would wrap each script tag in a div and use CSS to algin them how I want.

On a general note: I would not feel good about using such a script in my pages, since it more or less provides hackthebox and tryhackme the ability to do anything they want to to your page. Sadly it seems to me like they offer no other means of embedding these badges like via iframers.

Also just a small note: This project is about 11ty (eleventy), which is a static site generator (SSG) like Jekyll, but the projects are not related at all.

**Reply by CSpanias** [reply]
Hi @Snapstromegon , 

Thanks for your reply. I must ended up here through the forum, I never noticed that I was in another project's forum! Sorry about that!
